test/Conversion/TorchToLinalg/view.mlir

```diff
@@ -0,0 +1,39 @@
+// RUN: torch-mlir-opt <%s -convert-torch-to-linalg -split-input-file -verify-diagnostics | FileCheck %s
```

Comment:
I only see 2 tests in this file, but many e2e tests. Is there a reason that there is such a big difference?

Reply:
I'll make more unit tests, though I thought that a lot of the e2e tests were redundant.

Reply:
Okay, then let's delete the redundant e2e tests as well.